include/unifex/let_value_with_stop_token.hpp

```diff
@@ -95,8 +102,15 @@ class _stop_token_receiver<Receiver>::type {
   }
 
 private:
+  Operation* op_;
   inplace_stop_token stop_token_;
   UNIFEX_NO_UNIQUE_ADDRESS Receiver receiver_;
+
+  void cleanup() noexcept {
+    if (op_) {
```

Comment:
When is `op_` null?

Reply:
It's not and the check is redundant in this scenario. It's just that `op_` can potentially be `nullptr`. Removed.